src/components/scenes/Loans/LoanDashboardScene.tsx

```diff
@@ -169,14 +171,35 @@ export const LoanDashboardScene = (props: Props) => {
     <SceneWrapper background="theme" hasTabs={false}>
       <SceneHeader underline title={s.strings.loan_dashboard_title} />
       <EdgeText style={styles.textSectionHeader}>{s.strings.loan_active_loans_title}</EdgeText>
-      <FlatList
-        data={Object.values(loanAccounts)}
-        keyboardShouldPersistTaps="handled"
-        renderItem={renderLoanCard}
-        style={margin}
-        ListFooterComponent={renderFooter()}
-        keyExtractor={(loanAccount: LoanAccount) => loanAccount.id}
-      />
+      {Object.keys(loanAccountsMap).length === 0 ? (
+        <>
+          {isLoansLoading ? (
+            <Space isFill isGroupCenter isItemCenter horizontal bottom={2.5}>
```

Comment:
Suggesting ideas just cause:

```jsx
<Space fill center middle horizontal bottom={2.5}>
```

Maybe `start`, `center`, `end` for item alignment, and `beginning`, `middle`, `ending`. Though if we do this, then what's the point of `sideways`? Hmm, we should drop sideways if we include justification fields.

Alternatively, `alignLeft, alignRight, alignTop, alignBottom` to abstract away the concept of align v justify (which I think has always been confusing). Centering would get interesting, we could have `center` to center both directions, but if another vertical alignment is included then it will only center horizontally, or if a horizontal alignment is included then it'll only center vertically. In other words, `align*` props will always override `center` making things straight forward and expressive:

```jsx
<Space center> // Everything is aligned in the middle (very common usage)
<Space alignLeft center> // Only vertically centered because horizontal alignment is "left"
<Space alignTop center> // Only horizontally centered because vertical alignment is "right"
<Space alignTop alignLeft center> // This should log some warning that the center prop was completely overwritten
```

So with that.

```
<Space fill center horizontal bottom={2.5}>
``` 

Is all you would need, and it is easy to grok at. And no confusing sideways mode.

Reply:
Let's think about this more later, because I know for sure at least that having `center` override both directions isn't always what we want to do in the GUI. There are plenty of cases where we want to have `justify` and `align` have different values.